modules/tinymce/src/plugins/table/test/ts/browser/TableSectionApiTest.ts

```diff
@@ -110,11 +110,11 @@ describe('browser.tinymce.plugins.table.TableSectionApiTest', () => {
 
   const bodyMultipleChangesColumnContent = `<table>
 <tbody>
-<tr>
+<tr id="one">
```

Comment:
This is the only test here that didn't have id's on the row, so I suspect we hit this bug previously and forgot to log a Jira. For reference, this was the root issue this PR is attempting to fix as previously the id's would have been lost when changing the column type.